README.md

```diff
@@ -29,13 +33,15 @@ Create a virtual environment and download the required libraries.
 
 ## Usage
 
-Double click *run.bat* to find and use *config.txt* in the current folder. To use a different file, drag it onto *run.bat*.
+Drag an internet shortcut onto *run.bat* to create a config file (placed at *config.txt* by default) for the form. (This is equivalent to `run -c <shortcut> config.txt`.)
```

Comment:
Maybe write the full option in commands (`run --convert <shortcut> config.txt`).

Reply:
Improved in 16d3d868193533cef2c29f2552b30220eb6772e7.